draft-ietf-ppm-dap.md

```diff
@@ -152,6 +152,16 @@ input is ever seen in the clear by any aggregator.
 
 12:
 
+- Bump draft-irtf-cfrg-vdaf-08 to 12 {{!VDAF}}. (\*)
+
+- Add support for "asynchronous" aggregation, based on the Leader polling the
+  Helper for completion. (\*)
+
+- Update collection semantics to match the new aggregation semantics introduced
+  in support of asynchronous aggregation. (\*)
+
+- Clarify the requirements around report replay protection.
+
 - Remove the `max_batch_size` parameter of the fixed-size query type.
```

Comment:
Is this a breaking change, i.e., do we need `(\*)`?

Reply:
I don't think it "breaks wire compatibility" -- `max_batch_size` is specified as one of the task parameters, which have no defined serialization & are negotiated out-of-band from DAP.

(This is very likely to be a wire-breaking change to downstream specs which do define serialization for the task parameters, however; e.g. taskprov will presumably need to drop this parameter.)